src/components/EventInfo/Container.jsx

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components"
+import * as styleVars from "../variable"
+
+export const Container = styled.div`
+    background-color: ${styleVars.backgroundGradientEnd}
```

Comment:
Duplicated color variable

Reply:
This code must be in place since the removal of the `<Bg>` component as the background does not continue through the About-Sponsor pathways.

Here is an example of what happens when it is removed, I will look into a better fix but I believe this is the best approach for now.
![image](https://user-images.githubusercontent.com/83686967/143623190-d0cd6f1a-e339-43e2-9cf5-c483dfb1cc68.png)